packages/core/cli.js

```diff
@@ -56,7 +56,9 @@ if (userWantsGeneralHelp) {
 
 command
   .run(inputArguments, options)
-  .then(returnStatus => process.exit(returnStatus))
+  .then(returnStatus => {
+    returnStatus !== undefined ? process.exit(returnStatus) : process.exit();
```

Comment:
I might be missing something, but isn't this change equivalent to the previous version? That is, calling `process.exit(undefined)` should be identical to calling `process.exit()`, right?

Reply:
Those are equivalent, yeah.

Reply:
Ok so I can probably just remove that change and just remove the return value from `compile`. I'll close this and open a new PR.